README.md

```diff
@@ -20,25 +20,16 @@ time please let us know.
 
 ### How to submit your response
 
-Please submit your solution by emailing a link to [hiring@truss.works](mailto:hiring@truss.works).  More details on what we are looking for are in the problem READMEs linked below.
+Please submit your solution by emailing a link to [hiring@truss.works](mailto:hiring@truss.works). More details on what we are looking for are included in each problem description linked to below.
 
 ## Choose one problem to work on
 
 A README with specific instructions for each problem is linked below.
 
 ### The problem: [CSV normalization](CSV_README.md)
 
-Please write a tool that reads a CSV formatted file on `stdin` and
-emits a normalized CSV formatted file on `stdout`. For example, if
-your program was named `normalizer` we would test your code on the
-command line like this:
-
-```sh
-./normalizer < sample.csv > output.csv
-```
+This option involves writing a command-line program to process CSV data.
 
 ### The problem: [Display data from an API for human consumption](BROWSER_README.md)
 
-Please write a static webpage that loads data from a specified public API, and formats & displays that data in HTML. You may use libraries, frameworks and/or build tools as long as clear, simple instructions are provided for how to run them. Please ensure your solution can be easily viewed in the browser.
-
-You may use additional files to organize your code. You may also use frameworks and build tools as long as clear, simple instructions are provided for how to run them. Please ensure your solution can be easily viewed in the browser.
+This option involves creating a single page web application that loads data from an API.
```

Comment:
I like this!  I also wonder about moving the description out of the header.  Sometimes header links just take you to the header itself on the page.